Clarify Rover update notice for contract checks (#216)
apollographql/docs: src/content/studio/contracts.mdx

## Contract checks

> ⚠️ **Important:** To run contract checks via `rover subgraph check`, you must [update the Rover CLI](/rover/getting-started/#installation-methods) to v0.8.2 or later.
>

> > - **If you use Rover v0.8.0 or v0.8.1,** contract checks _do_ run, but the execution ignores any [blocking downstream variants](#failing-contract-checks) you've set.

Reviewer: Slight suggestion
```suggestion
> - **If you use Rover v0.8.0 or v0.8.1,** contract checks _do_ run, but Rover ignores any contract check failures in [blocking downstream variants](#failing-contract-checks) you've set.
```

That reason is that the backend and Studio UI will properly show the check's status in such cases as failed, even for checks initiated by Rover v0.8.0 and v0.8.1. The issue is purely a Rover bug, where it doesn't actually look at the check's status when it should be.
